Runtime/Scripts/msCoreAPI.cs

```diff
@@ -2945,8 +3034,75 @@ public Matrix4x4[] transforms
 
 #endregion
 
-internal struct SceneData {
-    #region internal
+
+
+#region PropertyInfo
+
+    public struct PropertyInfoData
```

Comment:
I need this to be public because of the regenerator we are building for Bay. This is built on top of MeshSync as a separate tool and requires access to the blender properties to change them and get updated meshes out of blender.

Reply:
Actually, I've changed it now and made the enums inside it public, that's all I really need.